README

```diff
@@ -21,6 +21,9 @@ the source tree.
 BUILDING FROM SOURCE
 ======================================================================
 
+Install dependencies
+  sudo apt install build-essential ca-certificates gcc make automake libtool libguestfs-tools libguestfs-dev libnbd-dev gettext libpcre2-dev libvirt-dev libxml2-dev libjansson-dev libosinfo-1.0-dev ocaml-findlib libvirt-ocaml-dev libguestfs-ocaml-dev libnbd-ocaml-dev
```

Comment:
The commit message is not clear - what do you mean by "AM_GNU_GETTEXT"? maybe you mean that the `gettext` package was missing?

But I think it will be better to use the equivalent of `dnf builddep` for apt based systems, installing the needed packages based on the package metadata. With this you get all the packages in most cases and there is no need to maintain this packages list.

If we go with actual package list, it should be sorted and formatted as one package per line:

    sudo apt install \
        build-essential \
        ca-certificates \
        gcc \
        ...

Reply:
IMHO better instructions would be just:
```
        apt-get build-dep virt-v2v
        apt-get install autoconf automake libtool-bin gettext
```